projects/storefrontapp-e2e-cypress/cypress/e2e/regression/user_access/otp-login.e2e-flaky.cy.ts

```diff
@@ -0,0 +1,131 @@
+          .its('response.statusCode')
+          .should('eq', 201);
+
+        cy.get('cx-verification-token-form').should('exist');
+        cy.get('cx-verification-token-form').should('be.visible');
+
+        cy.log('The email being sent is asynchronous, so waiting 10s');
+        cy.wait(10000);
+
+        const API_ENDPOINT =
+          'http://mail-ccv2.westeurope.azurecontainer.io:8025/api/v2/search';
+        cy.request({
+          method: 'GET',
+          url:
+            API_ENDPOINT + '?query=' + user.email + '&kind=to&start=0&limit=2',
+        }).then((response) => {
+          const subject =
+            '[Spartacus Electronics Site] Login Verification Code';
+          const verificationCodeEmailStartText =
+            'Please use the following verification code to log in Spartacus Electronics Site:</p>';
+          const lableP = '<p>';
+
+          const items = response.body.items;
+          const emailBody =
+            subject === items[0].Content.Headers.Subject[0]
+              ? items[0].Content.Body
+              : items[1].Content.Body;
+
+          const verificationCodeEmailStartIndex =
+            emailBody.indexOf(verificationCodeEmailStartText) +
+            verificationCodeEmailStartText.length;
+          const verificationCodeStartIndex =
+            emailBody.indexOf(lableP, verificationCodeEmailStartIndex) +
+            lableP.length;
+          const verificationCode = emailBody.substring(
+            verificationCodeStartIndex,
+            verificationCodeStartIndex + 8
+          );
+          cy.log('Extracted verification code: ' + verificationCode);
+
+          login.listenForTokenAuthenticationRequest();
+          cy.get('cx-verification-token-form form').within(() => {
+            cy.get('[formcontrolname="tokenCode"]')
+              .clear()
+              .type(verificationCode);
+            cy.get('button[type=submit]').click();
+          });
+          cy.wait('@tokenAuthentication')
+            .its('response.statusCode')
+            .should('eq', 200);
+        });
+      });
+    });
+
+    describe('Failed to Create OTP', () => {
+      beforeEach(() => {
+        cy.visit('/login');
+      });
+      it('should be not able to create OTP with invalid user data (CXSPA-6672)', () => {
+        listenForCreateVerificationToken();
+
+        cy.get('cx-otp-login-form form').within(() => {
+          cy.get('[formcontrolname="userId"]')
+            .clear()
+            .type('test.user@sap.coma');
+          cy.get('[formcontrolname="password"]').clear().type('1234');
+          cy.get('button[type=submit]').click();
+        });
+
+        cy.wait('@createVerificationToken')
+          .its('response.statusCode')
+          .should('eq', 400);
+
+        cy.get('cx-global-message').within(() => {
+          cy.get('span').contains('Email is not valid.');
+        });
+      });
+    });
+
+    describe('Chould can return back to login page from verify code page', () => {
```

Comment:
I guess you want to say this
```suggestion
    describe('Should go back to login page when click back button', () => {
```

Reply:
changed